CHANGELOG.md

```diff
@@ -29,6 +25,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 # Past Releases
 
+## [0.28.1] - 2021-09-10
```

Comment:
Don't forget to add this to the bottom of the file too!

Reply:
- [ ] Add version at bottom of changelog

Reply:
@kyleve: [This look right?](https://github.com/square/Blueprint/commit/e8cee52bff78d816580d49fea298bffc096ff199)

Reply:
Yep!